app/views/admin/base/dashboard.html.erb

```diff
@@ -1,8 +1,8 @@
 <div id="admin_header">
-
   <h1>Admin Dashboard</h1>
 </div>
 
-<h2>This is where the Platform Admin dashboard stuff will go.</h2>
 
-<%= link_to "Categories", admin_categories_path  %>
+<p><%= link_to "Categories", admin_categories_path %></p>
+<p><%= link_to "All Borrowers", admin_tenants_path %></p>
```

Comment:
I misunderstood your comment.  I thought you were saying it was already in the navbar and I kept looking for it.

Yeah I'll move it to the navbar.  The profile button has to be removed for the admin anyway so it will fit in perfect.